content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+---
+title: API Server Flow Control
+content_template: templates/concept
+min-kubernetes-server-version: v1.18
+---
+
+{{% capture overview %}}
+
+Controlling the behavior of the Kubernetes API server in an overload situation
+is a key task for cluster administrators. The{{< glossary_tooltip term_id="kube-apiserver" text="kube-apiserver" >}} has some controls
+available (i.e. the `--max-requests-inflight` and
+`--max-mutating-requests-inflight` command-line flags) to limit the amount of
+outstanding work that will be accepted, preventing a flood of inbound requests
+from overloading and potentially crashing the API server, but beyond that, the
+Priority and Fairness (or flow-control) API provides a configurable mechanism to
+ensure that in a high-load situation, forward progress will be made on the right
```

Comment:
i suppose this overview only covers the aspect of controlling bursty traffic in protection of API server, while another aspect of the feature is to provide a configurable requests __fairness__ among namespaces. we should note that it's also a mechnism of noise-proofing/isolation for namespaces/tenants